### PR TITLE
Require openkim-models 

### LINKED ISSUE #10: delete/close this repo/package in favor of openkim-models?
@jan-janssen Why are there two openkim models packages?  We should get rid of this one to avoid confusion, I think.

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge-admin, please rerender

**Review by ellio167 — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,31 +1,23 @@
...
-    - {{ compiler('fortran') }}
-    - cmake
-    - make
-    - vim
-    - pkg-config
   host:
-    - kim-api
   run:
-    - kim-api
+    - openkim-models
```
Comment (jan-janssen) [inline recipe/meta.yaml:20]:
```suggestion
    - openkim-models >=2021.01.28
```

**Comment by jan-janssen** [comment]
@conda-forge-admin, please rerender

**Comment by jan-janssen** [comment]
somehow the tests failed so I opened a new pull request https://github.com/conda-forge/openkimmodels-feedstock/pull/13

**Pull request closed without merging**